recipe/meta.yaml

```diff
@@ -25,13 +25,16 @@ source:
 
 build:
   number: 6
-  skip: true  # [osx or win]
+  skip: true  # [osx]
 
 outputs:
   - name: cuda-cudart_{{ target_platform }}
     build:
       noarch: generic
       binary_relocation: false
```

Comment:
Now that we know it was patchelf causing issues, perhaps it's time to remove this workaround?

Reply:
AFAIK the `patchelf` bug is not fixed: https://github.com/NixOS/patchelf/issues/492

So no don't think we should be removing that

Reply:
No, it's not fixed (I am fully aware), but we marked 0.18.0 as broken ([you did it](https://github.com/conda-forge/admin-requests/pull/746) 🙂) and there's no 0.18.1 yet. I'd assume we are safe for now, and be extra cautious when a new patchelf is out. 

We can even go further and pin patchelf globally, which I was surprised to see not done yet. We should consider it given its high impact.

Reply:
Would rather not open us to unnecessary risk with `patchelf`. This keeps us protected

We explored that before ( https://github.com/conda-forge/conda-smithy/issues/1737 ), but there wasn't a good option to pursue ( https://github.com/conda-forge/conda-smithy/issues/1737#issuecomment-1614409419 )

Reply:
I meant to add patchelf to conda-forge-pinning-feedstock, not conda-smithy. Is it not an option (and why)?

Reply:
Unclear on how that would help. `conda-forge-pinning` only affects dependencies in the `requirements` of recipes themselves

Not build tools like `patchelf` that are installed in `base`. This would require change to `conda-smithy` to work

Reply:
Thanks, John, I see. I think this is where it confused me.

I know for a fact that installing patchelf to host could help ignore the one installed by conda-smithy. But it requires the recipe maintainer to manually add patchelf to host, which is something awkward that we don't want.

Reply:
And there would still be a `patchelf` installed in `base` that `conda-build` would use. IOW the problem would still be there